stable/aws-cloudwatch-metrics/templates/daemonset.yaml

```diff
@@ -12,6 +12,10 @@ spec:
     metadata:
       labels:
         {{- include "aws-cloudwatch-metrics.selectorLabels" . | nindent 8 }}
+      {{- if .Values.podAnnotations }}
```

Comment:
as annotations exist on aws-for-fluent-bit chart. Maybe we should harmonize the variable
`{{- if .Values.annotations }}`